### ISSUE TITLE
Missing csv file in sprite_moving_platforms example

### ISSUE THREAD

**Issue by ChrisJefferson** [pvcraven/arcade#248]
## Bug Report

The example at `sprite_moving_platforms.html` doesn't run, because there is no "map.csv" distributed with arcade.

### Actual behavior:

```
python3 -m arcade.examples.sprite_moving_platforms
...
FileNotFoundError: [Errno 2] No such file or directory: 'map.csv'
```
### Expected behavior:
map.csv is found, game runs :)
### Steps to reproduce/example code:
```
python3 -m arcade.examples.sprite_moving_platforms
```
## Enhancement request:

Add an example map.csv

**Comment by ChrisJefferson** [comment]
sprite_tiled_map.py also seems to refer to map.csv.

**Comment by ChrisJefferson** [comment]
map_with_ramps_2.csv is also missing, from sprite_ramps.py (sorry for so many posts)

**Comment by BORN2LOSE** [comment]
Same issue when i install from pip on GNU/Linux.
For example, grab from Github and move to dir: 
```bash
wget https://raw.githubusercontent.com/pvcraven/arcade/master/arcade/examples/map.csv
sudo mv map.csv /usr/lib/python3.7/site-packages/arcade/examples
```

**Comment by pvcraven** [comment]
Closed with 2.0 release.
